src/views/pages/dashboard.ejs

```diff
@@ -16,6 +16,9 @@
 
       <div class="content">
         <div class="container-fluid">
+          <div id="twitch-player" class="text-center" style="display: none;">
+            <iframe src="https://player.twitch.tv/?channel=greatunihack&parent=hub.greatunihack.com" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe>
```

Comment:
We should make an issue to not hardcode the channel name

Reply:
:+1: #154